mne/beamformer/_lcmv.py

```diff
@@ -24,6 +24,16 @@
 from ..channels.channels import _contains_ch_type
 
 
+def _deprecate_picks(data_obj, picks):
+    if picks is not None:
+        warn('Specifying picks is deprecated and will be removed in 0.16. '
+             'Specifying the selection of channels in info and setting picks '
+             'to None will remove this warning.',
+             DeprecationWarning)
+
+        data_obj.info = pick_info(data_obj.info, picks)
```

Comment:
And my solution only works for data that is loaded into memory, so it's not so good.

But what you have here will not work properly because it makes a mismatch between the data and info, so another solution is needed.